### PR TITLE
Complete Redesign (Request by CozyPlanes)

### PR DESCRIPTION
**Pull request by jonasgeiler**
CozyPlanes asked me to completely redesign the site in the style of my DevRant-Stats site.
So here you are!

It might not be the best design but it's definitely a improvement!
Feel free to change some things if you want. I **probably** won't be angry :D

Here's a screenshot:
screenshot

I hope you enjoy!
Please leave a feedback,
Skayo

### CODE REVIEW

**Comment by Kimmax** [comment]
Looks very good to me
Could you possibly setup your github pages to preview this in a subfolder or something?
Otherwise I could do it myself tomorrow
Do you know the license of the header bg by any chance? Otherwise I'll drop dfox a quick mail and get that cleared
Thanks for your work!

File: README.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ Well, it is easier than you think.
 
 1. Open up your FAVORITE browser. We support way down until Netscape Navigator 7.20 ! (We have tried hard to achieve this.)
 2. Head up to https://devbanner.center
-3. You will be greeted by the beautiful, colorful frontend.
+3. You will be greeted by the new beautiful, colorful frontend by Skayo.
```
Comment (Kimmax) [inline README.md:42]: I'm fine with mentioning your directly for starters, as long as it's okay for you if we might drop that in a couple of weeks or so if the frontend did some major continued evolving.
We'll never drop you from the `Credits` section as long as we use your work tho (Probably not even after dropping all your work, if we should ever do that)

File: index.html [hunk 2]
```diff
@@ -1,51 +1,111 @@
...
+	<link href="https://fonts.googleapis.com/css?family=Comfortaa:400,700" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" media="screen" href="lib/AnimateCSS/animate.min.css" />
+	<link rel="stylesheet" type="text/css" media="screen" href="res/css/stylesheet.css" />
+	<link rel="stylesheet" type="text/css" media="screen" href="res/css/spinner.css" />
+</head>
+<body>
+<div class="content">
+	<div id="header">
+		<span id="header-before" style="background-color: #9e85a1;"></span>
+		<img id="header-img" class="img-fluid" src="res/images/logo_transparent.png">
```
Comment (Kimmax) [inline index.html]: Licensing  of the header image, as mentioned above

Comment (Kimmax) [inline index.html:21]: Licensing of the header image, as mentioned above

File: README.md (hunk 1 above)
Comment (jonasgeiler) [reply README.md:42]: That's okay haha

File: index.html (hunk 2 above)
Comment (jonasgeiler) [reply index.html:21]: Well devRantron uses it, DevRant-Stats uses it, you use it now.
Shouldn't be a problem!

**Review by cozyplanes — APPROVED** [review]

**Comment by cozyplanes** [comment]
Seems nice by now. Merging. Thanks, @Skayo !

**Comment by cozyplanes** [comment]
@Skayo @Kimmax Can you check the CSS? It is literally f###ed up... I wonder if I touched something wrong....

**Comment by cozyplanes** [comment]
I accidently made that revert PR, and it is closed. Sorry!

**Comment by cozyplanes** [comment]
https://devbanner.center <-- link

**Pull request merged**